website/i18n/en.json

```diff
@@ -143,7 +143,6 @@
       },
```

Comment:
Could you remove the `website/i18n/en.json` file from this PR? When it gets pushed up to master, it gets generated.